app/config/config_dev.yml

```diff
@@ -14,12 +14,20 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:   stream
-            path:   "%env(SYMFONY_LOG)%"
-            level:  info
+            type: stream
+            path: "%env(SYMFONY_LOG)%"
```

Comment:
@ogizanagi, what about to use a name like `LOG_URL` for consistency with others variables names (`DATABASE_URL`, `MAILER_URL`)?